apps/users/templates/users/pwreset_request.html

```diff
@@ -11,6 +11,16 @@
     <form method="post" class="listing-footer">
       {{ csrf() }}
       <fieldset>
+        {% if request.GET.migrated %}
```

Comment:
Relying on `?migrated` seems a little awkward to me. What about re-checking `user.fxa_id` when this view loads and passing in `user_has_migrated` (or whatever) to the view template? A user who is monkeying with a query string probably can't do anything malicious but it seems like bad practice.